MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+The current Maintainers Group for the KubeArchive Project consists of:
+
+* [Andrea Fernandez](https://github.com/mafh314), [Red Hat](https://www.redhat.com)
+* [Chris Beer](https://github.com/beerparty), [Red Hat](https://www.redhat.com)
+* [Greg Allen](https://github.com/ggallen), [Red Hat](https://www.redhat.com)
+* [Hector Martinez](https://github.com/rh-hemartin), [Red Hat](https://www.redhat.com)
```

Comment:
As we are going with accents maybe you prefer having the accents?
```suggestion
* [Héctor Martínez](https://github.com/rh-hemartin), [Red Hat](https://www.redhat.com)
```

Reply:
I don't use my accents in a professional setting, as there is a lot of programs that do not support them.